packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -323,8 +340,12 @@ const SchedulerForm: FC<{
                     <Tabs.Tab value="setup" ml="md">
                         Setup
                     </Tabs.Tab>
+                    {isSchedulerFiltersEnabled && isDashboard ? (
```

Comment:
I would even add 
```suggestion
                    {isSchedulerFiltersEnabled && isDashboard && dashboard ? (
```
but not blocking 💪

Reply:
good one, made the change